src/Commands/stubs/Auth/LoginController.stub

```diff
@@ -29,7 +28,7 @@ class LoginController extends Controller
             abort(401, 'Invalid credentials.');
         }
 
-        $token = Password::createToken($user);
+        $token = $user->createToken('login');
```

Comment:
You can use the same logic as the stub, where you don't create a variable that's not used